crates/rari-tools/src/sync_translated_content.rs

```diff
@@ -0,0 +1,433 @@
+            resolved_slug = concat_strs!("conflicting/", &resolved_slug).into();
+            if md_or_html_exists(&resolved_slug, doc.locale())? {
+                let mut hasher = md5::Md5::new();
+                hasher.update(doc.slug());
+                let digest = format!("{:x}", hasher.finalize());
+                // println!("digest: {}", digest);
+                resolved_slug = concat_strs!(&resolved_slug, "-", &digest).into();
+            }
+
+            status.conflicting = true;
+        } else {
+            return Err(ToolError::Unknown("Conflicting docs not found"));
+        }
+    }
+
+    status.redirect = Some((
+        build_url(doc.slug(), doc.locale(), PageCategory::Doc)?,
+        build_url(&resolved_slug, doc.locale(), PageCategory::Doc)?,
+    ));
+    if verbose {
+        println!(
+            "{}",
+            blue.apply_to(format!("redirecting {} → {}", doc.slug(), resolved_slug))
+        )
+    }
+
+    // Update the wiki history
+    update_wiki_history(
+        doc.locale(),
+        &[(doc.slug().to_string(), resolved_slug.to_string())],
+    )?;
+
+    if status.moved {
+        move_doc(doc, &resolved_slug)?;
+        // move the doc to the new location
+        // set the original slug in metadata of moved doc
+    }
+
+    Ok(status)
+}
+
+fn move_doc(doc: &Doc, target_slug: &str) -> Result<(), ToolError> {
+    let source_path = doc.path();
+    let target_directory = root_for_locale(doc.locale())?
+        .join(doc.locale().as_folder_str())
+        .join(url_to_folder_path(target_slug));
+    std::fs::create_dir_all(&target_directory)?;
+
+    // Write the new slug, store the old slug in `original_slug` metadata
+    let mut new_doc = doc.clone();
+    new_doc.meta.slug = target_slug.to_owned();
+    new_doc.meta.original_slug = Some(doc.slug().to_owned());
+    new_doc.write()?;
+
+    // Move the file with git
+    let output = exec_git_with_test_fallback(
+        &[
+            OsStr::new("mv"),
+            source_path.as_os_str(),
+            target_directory.as_os_str(),
+        ],
+        root_for_locale(doc.locale())?,
+    );
+
+    if !output.status.success() {
+        return Err(ToolError::GitError(format!(
+            "Failed to move files: {}",
+            String::from_utf8_lossy(&output.stderr)
+        )));
+    }
+
+    // If the source directory is empty, remove it with the fs api.
+    let source_directory = doc.full_path().parent().unwrap();
+    if source_directory
+        .read_dir()
+        .map(|mut dir| dir.next().is_none())
+        .unwrap_or(false)
+    {
+        std::fs::remove_dir(source_directory)?;
+    }
+
+    Ok(())
+}
+
+fn md_or_html_exists(slug: &str, locale: Locale) -> Result<bool, ToolError> {
+    let folder_path = root_for_locale(locale)?
+        .join(locale.as_folder_str())
+        .join(slug.to_lowercase());
+    let md_path = folder_path.join("index.md");
+    let html_path = folder_path.join("index.html");
```

Comment:
We no longer need to support `index.html` (cf. https://github.com/mdn/yari/pull/7575).

Reply:
Ok, I'll remove that